src/app.controller.ts

```diff
@@ -9,4 +9,12 @@ export class AppController {
   getHello(): string {
     return this.appService.getHello();
   }
+
+  @Post('bad-format')
+  async bad_format(@Body( ) data: any)
+  {
+    console . log ( 'this should not be legal');
+
+    return {hehe:  42};
```

Comment:
Codacy has a fix for the issue: [Replace `hehe:··42` with `·hehe:·42·`](https://app.codacy.com/gh/darklam/code-review-testing/pullRequest?prid=8068379)
```suggestion
    return { hehe: 42 };
```